web/src/click_handlers.ts

```diff
@@ -347,11 +348,17 @@ export function initialize(): void {
     $("body").on("click", ".message_edit_cancel", function (e) {
         const $row = $(this).closest(".message_row");
         message_edit.end_message_row_edit($row);
+        compose_tooltips.clean_up_existing_instances(
+            `edit_message:${$row.attr("data-message-id")}`,
+        );
         e.stopPropagation();
     });
     $("body").on("click", ".message_edit_close", function (e) {
         const $row = $(this).closest(".message_row");
         message_edit.end_message_row_edit($row);
+        compose_tooltips.clean_up_existing_instances(
+            `edit_message:${$row.attr("data-message-id")}`,
+        );
```

Comment:
Looks like `message_edit.end_message_row_edit()`, is the common call in both the cases, so let's move the cleanup to that method.